main.go

```diff
@@ -134,5 +134,9 @@ func loggingMiddleware(next http.Handler) http.Handler {
 
 // logRequest converts a request object into a proper logging event
 func logRequest(r *http.Request) {
+	// Do not log requests to the health endpoint
+	if r.RequestURI == "/health" {
```

Comment:
Hm... I think you shouldn't strip it. It's valuable to see that the app receives requests.

Reply:
My problem here was that I have configured the Docker healthcheck to poll it every second. So this ends up with quite a few entries.